min-image/runtimes/python/server.py

```diff
@@ -158,6 +158,14 @@ def start_container():
         # works for the process that calls it.
         os._exit(0)
 
+    # drop all capabilities
+    return_val = ol.drop_all_caps();
+    assert return_val == 0
+
+    # set CAP_SYS_ADMIN
+    return_val = ol.modify_caps(21, 1);
```

Comment:
Can we avoid hardcoding this?  Anyway we can expose some consts (like CAP_SYS_ADMIN) from our ol module to Python?

Reply:
Wait, aren't we always doing this, regardless of cmd line args?

Reply:
@tylerharter Right, for that, I was concerned that I would have to pass that boolean flag down the function calls. `enable-seccomp` bool is only needed in the beginning where it is "enabled" and then that's it. But since we would have to drop privileges at certain points in the code, we would need the `drop-capabilities` bool value to propagate down.

I left it as it is for now for testing while I thought of a good solution for it or if you have any suggestions.